apps/src/templates/projects/ProjectCard.story.jsx

```diff
@@ -1,120 +1,39 @@
 
 export default storybook => {
   storybook
     .storiesOf('ProjectCard', module)
     .addStoryTable([
       {
-        name: 'Project card in personal recent project widget',
-        description: 'Personal gallery widget should NOT show student display name or chevron.',
-        story: () => (
-          <ProjectCard
-            projectData={personalData}
-            currentGallery="personal"
-            hideActions={true}
-          />
-        )
-      },
-      {
-        name: 'Project card on personal gallery, not published on any other gallery',
-        description: 'Personal gallery should NOT show student display name and should show chevron, which on-click shows a box with publishing options. Publishing options should include Publish to Class Gallery and Publish to Public Gallery',
-        story: () => (
-          <ProjectCard
-            projectData={personalData}
-            currentGallery="personal"
-          />
-        )
-      },
-      {
-        name: 'Project card on personal gallery, published on class gallery',
-        description: 'Personal gallery should NOT show student display name and should show chevron, which on-click shows a box with publishing options. Publishing options should include Remove from Class Gallery and Publish to Public Gallery',
-        story: () => (
-          <ProjectCard
-            projectData={publishedToClassPersonal}
-            currentGallery="personal"
-          />
-        )
-      },
-      {
-        name: 'Project card on personal gallery, published on public gallery',
-        description: 'Personal gallery should NOT show student display name and should show chevron, which on-click shows a box with publishing options. Publishing options should include Publish to Class Gallery and Remove from Public Gallery',
-        story: () => (
-          <ProjectCard
-            projectData={publishedToPublicPersonal}
-            currentGallery="personal"
-          />
-        )
-      },
-      {
-        name: 'Project card on personal gallery, published on both public and class gallery',
-        description: 'Personal gallery should NOT show student display name and should show chevron, which on-click shows a box with publishing options. Publishing options should include Remove from Class Gallery and Remove from Public Gallery',
-        story: () => (
-          <ProjectCard
-            projectData={publishedToBothPersonal}
-            currentGallery="personal"
-          />
-        )
-      },
-      {
-        name: 'Project card on classroom gallery',
-        description: 'Classroom gallery should show student display name and should NOT show chevron',
+        name: 'Project card - public',
+        description: 'Project Gallery card used in the public gallery',
         story: () => (
           <ProjectCard
-            projectData={publishedBoth}
-            currentGallery="class"
+            projectData={defaultData}
+            currentGallery={'public'}
```

Comment:
string props don't need braces (i.e. `currentGallery="public"` instead of `currentGallery={"public"}`)